old_source/blog/001-how-extend-works.html.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: How @extend Works
 author: Natalie Weizenbaum
-date: 2013-11-22 16:57 PST
+# date: 2013-11-22 16:57 PST
```

Comment:
Probably revert these changes -- I think ideally we don't need to make any changes directly to this `old_source` directory.